ONSdigital/florence
# Feature/use page end point for creation

### What

- Change `_postContent` to use new Zebedee `/page` endpoint when creating page types `api_dataset_landing_page`
- Update `deletePage` method to use new Zebedee `/page` endpoint
- Add conditional to catch empty 204 responses (deletes) and resolve, rather than retry and display an error even when request is successful. 

### How to review

- Create `api_dataset_landing_page` page and check no errors, and that the dataset is also created in the database. 
- Delete `api_dataset_landing_page` from collection and check no errors and that dataset is removed from the database
- Test other content still works as expected by creating and deleting (from collections) other content types

### Who can review

Anyone.
